gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcDirectBidiStreamingCallable.java

```diff
@@ -54,11 +54,11 @@
 
   @Override
   public ApiStreamObserver<RequestT> bidiStreamingCall(
```

Comment:
I'm wondering if the returned typed should be updated as well. After this change it will only be used for streaming messages to the server. Might be good to rename it to something like `ClientStream`? Also, at some point in the future it would be nice to expose backpressure by adding the methods `isReady()` and `onReady(callable)`

Reply:
Why not BidiStream?